zadani/soft/style.css

```diff
@@ -15,29 +16,311 @@ body {
     font-family: 'Merriweather', serif;
     font-size: 16px;
     line-height: 1.6;
+    
 }
 
 
 /* HLAVICKA */
+.hlavicka {    
+    background-image: linear-gradient(rgba(0,0,0,0.3),rgba(0,0,0,0.3)), 
+                    url("../../images/island-hlavicka.jpg"); 
+    background-position: center;
+    background-size: cover;
+    min-height: 200px;
+    text-align: center;
+    padding: 5px;
+    margin: 0
+}
 
 .titulek {
-    font-family: 'Raleway', serif;
+    font-size: 50px;
+    font-family:'Raleway', serif;
+    color:white;
+    padding:20px;
+    margin:0px;
+}
+
+.perex {
+    font-size: 12px;
+    color:white;
+    padding:20px;
+    margin:0px;
+}
+
+.autor {
+    color: rgba(236, 233, 233, 0.836);
+    font-size: 12px;
+    font-style: italic;
+    padding:30px 20px 30px 20px;
+    position: relative;
 }
 
+.autor::after {
+    content: "";
+    width:18%;
+    border-top: 1px solid rgba(236, 233, 233, 0.863);
+    position: absolute;
+    left: 40%;
```

Comment:
Tady nám ta pozice někdy nevychází a element není přímo na středu. Je to proto, že využíváš relativní jednotky a odsazuješ o 40% od kraje prvku, takže spoléháš na to, že bude nějak široký, aby to vždy vyšlo do půlky. 
Bylo by lepší dát mu pevnou šířku v px a mohli bychom ho vycentrovat buď pomocí CSS kalkulačky (CSS calc) https://developer.mozilla.org/en-US/docs/Web/CSS/calc() - to ještě neznáme. 
Nebo využít transformací a dát mu `left: 50%;` a `transform: translate(-50%, 0);` - to by ho mělo vystředit vždy.